report-viewer/src/components/DistributionDiagram.vue

```diff
@@ -0,0 +1,146 @@
+<template>
```

Comment:
General remark: Please write 1-2 sentences of documentation for each component and view to briefly explain what it does.